### PR TITLE
Change landing page map legends and EQIP popper text

### LINKED ISSUE #91: Fix map legends and update text

### PR DESCRIPTION
**Pull request by ziyue5**
resolves #91 #105

### CODE REVIEW

File: src/components/ProgramDrawer.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ function EQIPCheckboxList({ setEQIPChecked, setShowPopUp }) {
 				return (
 					<Box key={category}>
 						<Typography sx={{ pl: 8 }}>
-							<strong>(6)(B) Practices</strong>
+							<strong>(6)(B) Plannings</strong>
```
Comment (navarroc) [inline src/components/ProgramDrawer.tsx:131]: Can you change this to Planning?

Comment (ziyue5) [reply src/components/ProgramDrawer.tsx:131]: so no "(6)(B)" in it, right?

Comment (navarroc) [reply src/components/ProgramDrawer.tsx:131]: (6)(B) Planning

Comment (navarroc) [reply src/components/ProgramDrawer.tsx:131]: (6)(A) Improvements

Comment (ziyue5) [reply src/components/ProgramDrawer.tsx]: Updated

**Review by sandeep-ps — APPROVED** [review]
Approving. I briefly tested this by visiting all pages. Thanks.

**Comment by navarroc** [comment]
Two things are needed to finish this:
1. Revert landing page maps to use the old legends (for now) - we need to wait for a longer term solution later
2. Comment out so the Crop Insurance page doesn't show

Then I think we can finish this release and tag it.

**Review by ywkim312 — APPROVED** [review]
It works well, approve

**Comment by sandeep-ps** [comment]
I see a change in font color and style in the label and the percentages. Was that intentional?

Current production: 
image
This branch: 
image

**Comment by ziyue5** [comment]
> I see a change in font color and style in the label and the percentages. Was that intentional?
> 
> Current production: image This branch: image

@sandeep-ps No, it's mainly because we are changing from image-hardcoded legend to our own legend component. So the fonts are slightly different.

**Comment by sandeep-ps** [comment]
> @sandeep-ps No, it's mainly because we are changing from image-hardcoded legend to our own legend component. So the fonts are slightly different. And the style should be the same as EQIP maps since EQIP maps already use our own legend component.

Okay. Thanks for the clarification. I'll approve this PR. You may want to have a follow-up task for sometime later to update the styles to match the Figma design.

**Review by sandeep-ps — APPROVED** [review]

**Comment by ziyue5** [comment]
> > @sandeep-ps No, it's mainly because we are changing from image-hardcoded legend to our own legend component. So the fonts are slightly different. And the style should be the same as EQIP maps since EQIP maps already use our own legend component.
> 
> Okay. Thanks for the clarification. I'll approve this PR. You may want to have a follow-up task for sometime later to update the styles to match the Figma design.

Thanks. No problem, I will do this soon.

**Pull request merged**